### PR TITLE
Update Mellanox/rdmamap dependency with the newest fixes

### PR DESCRIPTION
**Pull request by Mmduh-483**
Update the rdmamap with the newest bug fix of the rdma
device name

### CODE REVIEW

**Comment by Mmduh-483** [comment]
@zshi-redhat @ahalim-intel Can you please review it?

File: glide.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ import:
   version: ^0.3.0
 - package: github.com/vishvananda/netlink
 - package: github.com/Mellanox/rdmamap
+  vcs: git
+  version: master
```
Comment (ahalim-intel) [inline glide.yaml:20]: Can we avoid version locked to master? It's as good as not specifying a version at all.

**Comment by zshi-redhat** [comment]
@adrianchiris is there any new feature introduced in this update ? other than the name fix.

File: glide.yaml (hunk 1 above)
Comment (moshe010) [reply glide.yaml:20]: yes we should version it and update the commit

**Comment by moshe010** [comment]
> @adrianchiris is there any new feature introduced in this update ? other than the name fix.
No new features, just the name fix

File: glide.yaml (hunk 1 above)
Comment (Mmduh-483) [reply glide.yaml]: @ahalim-intel updated with version v1.0.0 of rdmamap

**Comment by adrianchiris** [comment]
@zshi-redhat the update of rdmamap contains two changes:
1. Name comparison for RDMA devices
2. support for go mod (there was no dependency management for the project)

we see other changes as one of rdmamap dependent packages got updated to a release tag version due 2. (github.com/jaypipes/ghw)

**Comment by moshe010** [comment]
can we merged this please?

**Pull request merged**